test/windows/acceptance/logstash_release_acceptance.ps1

```diff
@@ -0,0 +1,71 @@
+# Created By: Gabriel Moskovicz
+#
+# To be run on Jenkins
+#
+# Requirements to run the test:
+# 
+# - Powershell 4
+# - Windows 7 or newer
+# - Java 7 or newer
+
+$LS_CONFIG="test.conf"
+$LS_BRANCH="1.5"
```

Comment:
re: environment, I prefer flags (especially in scripts) rather than environment variables. Thoughts on having this set via flag/options instead?